### PR TITLE
[WIP]トップページ

### PR DESCRIPTION
**Pull request by shimada-code**
WHY
トップページを作るため

WHAT
トップページを確認するため

### CODE REVIEW

File: app/assets/stylesheets/modules/main.scss [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+  background-position: center;
+  background-size: cover;
+  padding-left: 100px;
+  &__Title {
+    font-size: 27px;
+    font-weight: bold;
+    color: #fff;
+    padding-top: 100px;
+    text-shadow: 0 0 5px rgba(0, 0, 0, 0.4);
+    color: #fff;
```
Comment (Nagisa9029) [inline app/assets/stylesheets/modules/main.scss:115]: > __[stylelint]__ Unexpected duplicate "color" ([view](https://sider.review/gh/repos/276010913/pulls/3#issue-4116978))
>
> <details>
>
> | Rule | Severity |
> | ---- | -------- |
> | `declaration-block-no-duplicate-properties` | `error` |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/13.6.1/lib/rules/declaration-block-no-duplicate-properties>
>
> You can [close this issue](https://sider.review/gh/repos/276010913/pulls/3#issue-4116978) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

重複指定になっています

File: app/assets/stylesheets/modules/main.scss [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  width: 100%;
+  height: 560px;
+  position: relative;
+  &__Title {
+    font-size: 26px;
+    text-align: center;
+    margin-bottom: 25px;
+    padding-bottom: 15px;
+    font-weight: bold;
+    text-align: center;
```
Comment (Nagisa9029) [inline app/assets/stylesheets/modules/main.scss:47]: > __[stylelint]__ Unexpected duplicate "text-align" ([view](https://sider.review/gh/repos/276010913/pulls/3#issue-4116977))
>
> <details>
>
> | Rule | Severity |
> | ---- | -------- |
> | `declaration-block-no-duplicate-properties` | `error` |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/13.6.1/lib/rules/declaration-block-no-duplicate-properties>
>
> You can [close this issue](https://sider.review/gh/repos/276010913/pulls/3#issue-4116977) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

銃封指定になっています

File: app/assets/stylesheets/modules/header.scss [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        }
+      }
+    }
+  }
+}
+.FloatBtn {
+  position: relative;
+  height: 112px;
+  width: 120px;
+  position: fixed;
```
Comment (Nagisa9029) [inline app/assets/stylesheets/modules/header.scss:94]: > __[stylelint]__ Unexpected duplicate "position" ([view](https://sider.review/gh/repos/276010913/pulls/3#issue-4116976))
>
> <details>
>
> | Rule | Severity |
> | ---- | -------- |
> | `declaration-block-no-duplicate-properties` | `error` |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/13.6.1/lib/rules/declaration-block-no-duplicate-properties>
>
> You can [close this issue](https://sider.review/gh/repos/276010913/pulls/3#issue-4116976) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

同じクラス内に【position】プロパティが２つ存在しています

File: app/assets/stylesheets/modules/main.scss (hunk 1 above)
Comment (shimada-code) [reply app/assets/stylesheets/modules/main.scss:115]: 修正しました

File: app/assets/stylesheets/modules/main.scss (hunk 2 above)
Comment (shimada-code) [reply app/assets/stylesheets/modules/main.scss:47]: 修正しました

File: app/assets/stylesheets/modules/header.scss (hunk 3 above)
Comment (shimada-code) [reply app/assets/stylesheets/modules/header.scss:94]: 修正しました

**Comment by shimada-code** [comment]
toppage1
cc1
toppage2
cc2
toppage3
cc3
toppage4
cc4
toppage5
<a href="https://gyazo.com/1dc42bd8765f59a873dca783ec91f3f2">Image from Gyazo</a>
toppage6
<a href="https://gyazo.com/b7ad6051a8175ed2746cd9d59aef6bec">Image from Gyazo</a>
toppage7
<a href="https://gyazo.com/3d19ec5dde55794926e73bc9c005706a">Image from Gyazo</a>

File: app/views/layouts/application.html.haml [hunk 4]
```diff
@@ -8,5 +8,95 @@
...
+          = link_to image_tag("topimage/logo.png"), "#" 
+        .Headfield__Top__Searchbox
+          = text_field :keyword,'', class: "Headfield__Top__Searchbox--input",placeholder: "キーワードから探す"
+          .Headfield__Top__Searchbox--icon
+            = image_tag ("topimage/icon-search 1.png"), class: "circle"
+      .Headfield__Bottom
+        .Headfield__Bottom__left
+          .Headfield__Bottom__left__kind
+            = link_to "カテゴリー", "#", class: "category"
+            -# .tree.displayNone
```
Comment (shingoteshima) [inline app/views/layouts/application.html.haml:25]: コメントアウト不要であれば消しましょう。

Comment (shimada-code) [reply app/views/layouts/application.html.haml]: 修正しました

**Comment by shingoteshima** [comment]
キャプチャありがとうございます！
LGTMです！！

**Pull request merged**